scripts/ci/generate_ci_job.py

```diff
@@ -37,11 +38,13 @@ def main(argv=sys.argv[1:]):
     add_argument_os_name(parser)
     add_argument_os_code_name(parser)
     add_argument_arch(parser)
+
```

Comment:
The extra space follows the pattern used in the other CI jobs, separating positional from optional arguments.